junction/urls.py

```diff
@@ -40,7 +39,7 @@
     url(r'^faq/$', TemplateView.as_view(template_name='static-content/faq.html',), name='faq-static'),
 
     # Proposals as conference home page. TODO: Needs to be enhanced
```

Comment:
@kracekumar `--` was added to identify the conference view, which doesn't exist at the moment. The conference view, right now just redirect to it's proposal view.

More explaination:

Routes of the form `/<conference>/proposals` works fine, but when you just do `/<conference>/` where `<conference>` is a slug, the urls of the form `/about/` or `/privacy/` can be easily confused as a conference name. `/<conference>--` pattern was added just for the sake of recognising a conference homepage.